Source/missiles.cpp

```diff
@@ -2843,6 +2843,11 @@ void MI_Dummy(int i)
 	return;
 }
 
+void MI_Dummy2()

```

Comment:
```suggestion
void dummy_nop_used_in_dx_init()
```

Also, move this to `dx.cpp` for now, to follow the convention used in `multi.cpp`:

```
Source/multi.cpp:		dummy_nop_used_in_NetInit();
Source/multi.cpp:void dummy_nop_used_in_NetInit()
Source/multi.h:void dummy_nop_used_in_NetInit();
```

Reply:
I wasn't 100% sure how to handle this because the original function `break_exception` appears to be part of appfat/fault.cpp. Not sure how it got stubbed to missiles.

Reply:
Most likely the body of `break_exception` was using `#ifdef _DEBUG`, so without debug defined, the body was simply nop.